flow-typed/npm/jest_v23.x.x.js

```diff
@@ -0,0 +1,1104 @@
+// flow-typed signature: ad251f3a3446f6ab4e6691a94e701cad
```

Comment:
You are removing many of the flow-typed files but adding this one. Is that intentional?

Reply:
Yes, all the others seem to be installed automatically when running `flow-typed install` but they're not actually needed, as if they were development dependencies. I still haven't found a way to filter them out. I got this tip from @juanpicado